core/Db/Schema/Mysql.php

```diff
@@ -334,6 +334,24 @@ public function getTablesCreateSql()
                                       ) $tableOptions
             ",
 
+            'archiving_metrics'     => "CREATE TABLE {$prefixTables}archiving_metrics (
+                                      metadataid BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+                                      idarchive BIGINT UNSIGNED NULL,
```

Comment:
```suggestion
                                      idarchive BIGINT UNSIGNED NOT NULL,
```

There should never be a line in this table without an `idarchive`, even if the archive itself could have already been deleted.